wrappers/android/librealsense/src/main/java/com/intel/realsense/librealsense/StreamFormat.java

```diff
@@ -32,9 +32,9 @@ public enum StreamFormat {
     Z16H(28),
     FG(29),
     Y411(30),
-    Y16I(31);
+    COMBINED_MOTION(31);
```

Comment:
Was there a Y16I format at some point?
If changing this file, I think that Z16H is unused anymore

Reply:
> Was there a Y16I format at some point?

There was (though I don't think it was released) -- and then was removed. The `rs2_stream` enum no longer has it, so I made sure this reflected the current state of the enum.

We cannot go and remove the old enum values (which **were** released).